scripts/cnv_wdl/germline/cnv_germline_case_scattered_workflow.wdl

```diff
@@ -196,16 +198,16 @@ workflow CNVGermlineCaseScatteredWorkflow {
 
     output {
         Array[File] preprocessed_intervals = CNVGermlineCaseWorkflow.preprocessed_intervals
-        Array[Array[File]] read_counts_entity_id = CNVGermlineCaseWorkflow.read_counts_entity_id
-        Array[Array[File]] read_counts = CNVGermlineCaseWorkflow.read_counts
-        Array[File] contig_ploidy_calls_tars = CNVGermlineCaseWorkflow.contig_ploidy_calls_tar
-        Array[Array[Array[File]]] gcnv_calls_tars = CNVGermlineCaseWorkflow.gcnv_calls_tars
-        Array[Array[File]] gcnv_tracking_tars = CNVGermlineCaseWorkflow.gcnv_tracking_tars
-        Array[Array[File]] genotyped_intervals_vcf = CNVGermlineCaseWorkflow.genotyped_intervals_vcf
-        Array[Array[File]] genotyped_segments_vcf = CNVGermlineCaseWorkflow.genotyped_segments_vcf
-        Array[Array[File]] qc_status_files = CNVGermlineCaseWorkflow.qc_status_files
-        Array[Array[String]] qc_status_strings = CNVGermlineCaseWorkflow.qc_status_strings
-        Array[Array[File]] denoised_copy_ratios = CNVGermlineCaseWorkflow.denoised_copy_ratios
+        Array[File] read_counts_entity_id = flatten(CNVGermlineCaseWorkflow.read_counts_entity_id)
+        Array[File] read_counts = flatten(CNVGermlineCaseWorkflow.read_counts)
+        Array[File] sample_contig_ploidy_calls_tars = flatten(CNVGermlineCaseWorkflow.sample_contig_ploidy_calls_tars)
+        Array[File] gcnv_calls_tars = flatten(CNVGermlineCaseWorkflow.gcnv_calls_tars)
```

Comment:
Does it make sense to flatten some of these quantities?

Reply:
You're right I shouldn't flatten `sample_contig_ploidy_calls_tars` and `gcnv_calls_tars`, but the rest are sample-wise and will correspond to the size/order of the input bams